operator/roles/kiali-deploy/templates/openshift/role.yaml

```diff
@@ -130,6 +130,26 @@ rules:
   - list
   - patch
   - watch
+- apiGroups: ["authentication.maistra.io"]
```

Comment:
these new permissions need to go in the operator's own role - otherwise the operator will not be able to install kiali.

So add to this file in this spot to remain consistent: https://github.com/kiali/kiali/blob/master/operator/deploy/role.yaml#L260

Reply:
I wonder ( @jmazzitelli , @lucasponce ) : are these changes with explicit mention to maistra fine even in a non-maistra setup, but still on openshift ?
I guess it's still going to work fine when "rbac.maistra.io" isn't anything on the cluster, but prefer to ask

Reply:
yeah, these api groups don't have to exist. they will be silently ignored. At least, that's what I have seen.